content/en/docs/concepts/glossary.md

```diff
@@ -201,6 +201,19 @@ series with [Metadata](#metadata). See [Metric].
 
 Short form for [OpenCensus](#opencensus).
 
+### Observability backend
+
+The component of an observability platform that is responsible for receiving,
+processing, storing, and querying telemetry data. Examples include open source
+tools like [Jaeger] and [Prometheus], as well as commercial offerings.
+OpenTelemetry is not an observability backend.
+
+### Observability frontend
+
+The component of an observability platform that provides user interfaces for
+visualizing and analyzing telemetry data. It is often part of an observability
+backend.
```

Comment:
```suggestion
The component of an observability platform that provides user interfaces for
visualizing and analyzing telemetry data. It can often be often a part of an
observability backend, particularly when considering commercial offerings.
```